content/en/apps/guides/hosting/requirements.md

```diff
@@ -10,6 +10,11 @@ relatedContent: >
   core/guides/docker-setup
 ---
 
+
+{{% alert title="Note" %}}
+For production CHT deployments, Linux is recommended, with [Ubuntu](https://ubuntu.com/server) the most commenly used. For CHT development, Linux or macOS may be used. Windows can be used for either, but without recommendation.
```

Comment:
```suggestion
For production CHT deployments, Linux is recommended, with [Ubuntu](https://ubuntu.com/server) the most commonly used. For CHT development, Linux or macOS may be used. Windows can be used for either, but without recommendation.
```

Reply:
`commenly` isn't even a word!  Thanks for spellcheck! :sweat_smile: